src/components/organisms/AssetActions/Pool/Remove.tsx

```diff
@@ -180,7 +180,6 @@ export default function Remove({
 
     const amountPoolShares = (Number(e.target.value) / 100) * Number(poolTokens)
     setAmountPoolShares(`${amountPoolShares}`)
-    calculateAmountOfOceansRemoved(`${amountPoolShares}`)
```

Comment:
We have pretty much the same logic for `handleMaxButton` should we remove `calculateAmountOfOceansRemoved` from there too?

Reply:
yes